mail/getmail6/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        getmail6 getmail6 6.19.03 v
+github.tarball_from tarball
+
+revision            0
+categories          mail python
+platforms           {darwin any}
+maintainers         wolfsden.cz:~+macports
+license             GPL-2
+description         extensible mail retrieval system with POP3, IMAP4, SSL support
+long_description    getmail6 is a flexible, extensible mail retrieval          \
+                    system with support for POP3, IMAP4, SSL variants of both, \
+                    maildirs, mboxrd files, external MDAs, arbitrary message   \
+                    filtering, single-user and domain-mailboxes, and many      \
+                    other useful features.
+supported_archs     noarch
+
+conflicts           getmail
+
+homepage            https://getmail6.org
+
+checksums           rmd160  992edce4932d14b4fdd1ef84fa04c8040a6bad5d \
+                    sha256  b46a257bd221b7ee0e55f4f8e84e7a2fee8c05ec3b61c08298c3605df250a14a \
+                    size    215166
+
+patchfiles          0001-pop3-Support-use_xoauth2.patch
```

Comment:
Where does this patchfile come from? We prefer not to carry around patches forever so if you created this patch, please submit it to the developers for inclusion in their next release. Please add a URL at the top of the patchfile showing either where you submitted it to upstream or where you got it from.